.github/workflows/haskell-ci.yml

```diff
@@ -214,14 +190,15 @@ jobs:
           touch cabal.project.local
           echo "packages: ${PKGDIR_brick}" >> cabal.project
           echo "package brick" >> cabal.project
-          echo "    ghc-options: -Werror=missing-methods" >> cabal.project
+          echo "    ghc-options: -Werror=missing-methods -Werror=missing-fields" >> cabal.project
+          if [ $((HCNUMVER >= 90400)) -ne 0 ] ; then echo "package brick" >> cabal.project ; fi
+          if [ $((HCNUMVER >= 90400)) -ne 0 ] ; then echo "    ghc-options: -Werror=unused-packages" >> cabal.project ; fi
```

Comment:
Recent version of haskell-ci adds this warning, which causes build error if there are unused dependencies in cabal file - for that reason I cleaned them up in cabal file to make fix build failures caused by this warning.